tests/mocha/unit-tests/sagas/coreFonts.spec.js

```diff
@@ -1,22 +1,73 @@
-import { call, put } from 'redux-saga/effects'
-import { getFilesFromGitHub } from '../../../../src/assets/js/react/sagas/coreFonts'
+import { channel } from 'redux-saga'
+import { call, fork, put, take } from 'redux-saga/effects'
+import {
+  getDownloadFonts,
+  getFilesFromGitHub,
+  watchDownloadFonts,
+} from '../../../../src/assets/js/react/sagas/coreFonts'
+import {
+  addToConsole,
+  addToRetryList,
+  currentDownload,
+  downloadFontsApiCall,
+  DOWNLOAD_FONTS_API_CALL,
+  GET_FILES_FROM_GITHUB_FAILED,
+} from '../../../../src/assets/js/react/actions/coreFonts'
 import * as api from '../../../../src/assets/js/react/api/coreFonts'
-import { GET_FILES_FROM_GITHUB_FAILED } from '../../../../src/assets/js/react/actions/coreFonts'
 
 describe('Sagas coreFonts -', () => {
 
   describe('getFilesFromGitHub()', () => {
-    const getFilesFromGitHubGenerator = getFilesFromGitHub()
+    const gen = getFilesFromGitHub()
 
     it('should check that saga asks to call the API for getFilesFromGitHub', () => {
-      expect(getFilesFromGitHubGenerator.next().value).to.be.eql(call(api.apiGetFilesFromGitHub))
+      expect(gen.next().value).to.deep.eql(call(api.apiGetFilesFromGitHub))
     })
 
     it('should check that saga handles correctly to the failure of getFilesFromGitHub API call', () => {
-      expect(getFilesFromGitHubGenerator.throw('failed').value).to.be.eql(put({
+      expect(gen.throw('failed').value).to.deep.eql(put({
         type: GET_FILES_FROM_GITHUB_FAILED,
         payload: 'failed'
       }))
     })
   })
+
```

Comment:
@debbieellis7 can you review these saga tests for the font download functionality please and let me know if everything is in order, or if modifications are needed?